index.html

```diff
@@ -931,6 +931,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg</code></td>
   </tr>
+  <tr><th> Uptime Robot status: </th>
+    <td><img src='https://img.shields.io/uptimerobot/status/u956-afus321g565fghr519.svg?maxAge=2592000' alt=''/></td>
```

Comment:
This example looks like the format used for Uptime Robot "full API keys". You should instead use a "monitor-specific API key" that looks like `m775661887-da53aab04d8b7fe0ed74e653` (`mxxxxx` rather than `uxxxxx` at the start) as monitor-specific API keys only allow read-only access.